data/sql/updates/pending_db_world/rev_1681662634958347500.sql

```diff
@@ -0,0 +1,11 @@
+--
+DELETE FROM `creature_template_movement` WHERE `CreatureId` = 18708;
+INSERT INTO `creature_template_movement` (`CreatureId`, `Ground`, `Rooted`) VALUES
+(18708, 1, 1);
```

Comment:
Disable Gravity as well
```suggestion
INSERT INTO `creature_template_movement` (`CreatureId`, `Ground`, `Rooted`, `Flight`) VALUES
(18708, 1, 1, 1);
```

Reply:
odd..